src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -323,7 +323,7 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="false">
```

Comment:
Binding client should be completely removed

Reply:
As all lighting devices come under Simple device types and they are not Client device types .. we should have a binding server as a mandatory cluster and the Binding client shouldn't be there,
So here server should be set to true

Reply:
Binding server is mandatory when device types is Simple and Client. Here Occupancy Sensing client cluster is optional. So I set Binding client turned off and user CANNOT turn it on, while Binding server turned off but user CAN turn it on.